hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/RouterMetrics.java

```diff
@@ -112,6 +116,10 @@ private RouterMetrics() {
     getClusterMetricsLatency =
         registry.newQuantiles("getClusterMetricsLatency",
             "latency of get cluster metrics", "ops", "latency", 10);
+
+    getClusterNodesLatency =
+            registry.newQuantiles("getClusterNodesLatency",
```

Comment:
Yetus seems not to be running the checkstyle.
Can you run it by hand and fix indentation?

CC @ayushtkn any idea on why the checkstyle is not being ran?

Reply:
no problem, I'll fix the indentation.

Reply:
It did ran @goiri for the latest 14th build, for the 5th the results aren't there now
Branch:
https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4274/14/artifact/out/buildtool-branch-checkstyle-hadoop-yarn-project_hadoop-yarn_hadoop-yarn-server_hadoop-yarn-server-router.txt
Patch:
https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4274/14/artifact/out/buildtool-patch-checkstyle-hadoop-yarn-project_hadoop-yarn_hadoop-yarn-server_hadoop-yarn-server-router.txt

Both have same number hence 0

Reply:
Interesting, the Yetus report is not showing and is giving a +1.